engine-rest/engine-rest-openapi/src/main/templates/models/org/camunda/bpm/engine/rest/dto/runtime/task/TaskQueryDto.ftl

```diff
@@ -0,0 +1,655 @@
+                [user guide](${docsUrl}/user-guide/process-engine/expression-language/#internal-context-functions)
+                for more information on available functions." />
+  
+    <@lib.property
+        name = "assigned"
+        type = "boolean"
+        defaultValue = "false"
+        desc = "If set to `true`, restricts the query to all tasks that are assigned." />
+  
+    <@lib.property
+        name = "unassigned"
+        type = "boolean"
+        defaultValue = "false"
+        desc = "If set to `true`, restricts the query to all tasks that are unassigned." />
+  
+    <@lib.property
+        name = "taskDefinitionKey"
+        type = "string"
+        desc = "Restrict to tasks that have the given key." />
+  
+    <@lib.property
+        name = "taskDefinitionKeyIn"
+        type = "array"
+        itemType = "string"
+        desc = "Restrict to tasks that have one of the given keys. The keys need to be in a
+                comma-separated list." />
+  
+    <@lib.property
+        name = "taskDefinitionKeyLike"
+        type = "string"
+        desc = "Restrict to tasks that have a key that has the parameter value as a substring." />
+  
+    <@lib.property
+        name = "name"
+        type = "string"
+        desc = "Restrict to tasks that have the given name." />
+  
+    <@lib.property
+        name = "nameNotEqual"
+        type = "string"
+        desc = "Restrict to tasks that do not have the given name." />
+  
+    <@lib.property
+        name = "nameLike"
+        type = "string"
+        desc = "Restrict to tasks that have a name with the given parameter value as substring." />
+  
+    <@lib.property
+        name = "nameNotLike"
+        type = "string"
+        desc = "Restrict to tasks that do not have a name with the given parameter
+                value as substring." />
+  
+    <@lib.property
+        name = "description"
+        type = "string"
+        desc = "Restrict to tasks that have the given description." />
+  
+    <@lib.property
+        name = "descriptionLike"
+        type = "string"
+        desc = "Restrict to tasks that have a description that has the parameter
+                value as a substring." />
+  
+    <@lib.property
+        name = "priority"
+        type = "integer"
+        format = "int32"
+        desc = "Restrict to tasks that have the given priority." />
+  
+    <@lib.property
+        name = "maxPriority"
+        type = "integer"
+        format = "int32"
+        desc = "Restrict to tasks that have a lower or equal priority." />
+  
+    <@lib.property
+        name = "minPriority"
+        type = "integer"
+        format = "int32"
+        desc = "Restrict to tasks that have a higher or equal priority." />
+  
+    <@lib.property
+        name = "dueDate"
+        type = "string"
+        format = "date-time"
+        desc = "Restrict to tasks that are due on the given date. By default*, the date must have the
```

Comment:
@yanavasileva, I forgot to add links to the default format in this file. Please reopen the ticket once you review it, so I can add `[default](${docsUrl}/reference/rest/overview/date-format/)`.